pkgs/tools/misc/zoxide/default.nix

```
@@ -15,15 +23,27 @@ rustPlatform.buildRustPackage {
     owner = "ajeetdsouza";
     repo = "zoxide";
     rev = "v${version}";
-    sha256 = "1ghdal6pqkp56rqawhj26ch1x4cvnjj032xz3626aiddqgn134zj";
+    sha256 = "143lh94mw31pm9q7ib63h2k842g3h222mdabhf25hpb19lka2w5y";
   };
 
+  # fish needs a writable HOME for whatever reason
+  preCheck = "export HOME=$(mktemp -d)";
```

```suggestion
  preCheck = ''
    export HOME=$(mktemp -d)
  '';
```

Usually this is written like this to support additions without changing to much or creating merge conflicts.

---

Fair enough. I don't expect this to be necessary, but you're right. Might as well.